### PR TITLE
fix for patch updateAttributes should be patchAttributes

### PR DESCRIPTION
**Pull request by hamzahio**
#208 fix for the problem.

### CODE REVIEW

File: lib/deserialize.js [hunk 1]
```diff
@@ -18,7 +18,8 @@ module.exports = function (app, options) {
 
     var regexs = [
       /^create$/,
-      /^updateAttributes$/
+      /^updateAttributes/,
```
Comment (digitalsadhu) [inline lib/deserialize.js]: Any reason you removed the $ at the end of updateAttributes?

File: package.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "loopback-component-jsonapi",
+  "version":"0.0.1",
```
Comment (digitalsadhu) [inline package.json]: We use semantic release which dynamically creates the correct version number on publish so we purposely don't keep this in package.json

https://github.com/semantic-release/semantic-release

Comment (hamzahio) [inline package.json]: Good no problem I just put it for using it in my project .

File: lib/deserialize.js (hunk 1 above)
Comment (hamzahio) [inline lib/deserialize.js]: I remove by mistake

**Comment by digitalsadhu** [comment]
closing in favour of #213 (where i have fixed and squashed)

**Pull request closed without merging**